nixbang

```diff
@@ -37,8 +37,13 @@ def main():
         print(HELP)
         return
     options = read_file(target)
-    command = '%s %s' % (options['command'], target)
+    args = ''
+    if len(sys.argv) > 1:
+       args = ' '.join(sys.argv[2:])
+
+    command = '%s %s %s' % (options['command'], target, args)
     subprocess.call(['nix-shell', '--command', command,
+                     '--args', args,
```

Comment:
Oh, yes woops. I was doing it with that first and changed it. Fixed in an updated commit! Thank you.